pygen/pygen_src/riscv_instr_pkg.py

```diff
@@ -24,7 +24,7 @@ class mem_region_t:
     xwr = auto()
 
 
-class satp_mode_t(Enum):
+class satp_mode_t(IntEnum):
```

Comment:
it doesn't seem like it, but switching everything over to IntEnum will not affect anything throughout the generator, correct?

Reply:
No it doesn't affect , and I've also confirmed this with @aneels3 that switching to `IntEnum` is a don't care for their implementation.

Reply:
sounds good to me!